R/vars-select.R

```diff
@@ -441,8 +441,11 @@ walk_data_tree <- function(expr, data_mask, context_mask, colon = FALSE) {
     literal = expr,
     symbol = eval_sym(as_string(expr), data_mask, context_mask, colon = colon),
     `(` = walk_data_tree(expr[[2]], data_mask, context_mask, colon = colon),
+    `!` = ,
     `-` = eval_minus(expr, data_mask, context_mask),
     `:` = eval_colon(expr, data_mask, context_mask),
+    `|` = eval_or(expr, data_mask, context_mask),
```

Comment:
I think we should probably provide || and && implementations that provide an informative error.

Reply:
Good idea.